src/services/command/command.ts

```diff
@@ -10,6 +10,11 @@ import type { Context } from "./context.js";
 
 export const AvailableCommands = ["sync", "list", "login", "logout", "whoami", "version"] as const;
 
+// temporarily remove deploy command from available commands while we're working on it
+if (!process.env["GGT_DEPLOY_PREVIEW"]) {
```

Comment:
Only add it if the env var is set:

```suggestion
if (process.env["GGT_DEPLOY_PREVIEW"]) {
```